Source/Icebreaker/Helpers/UserInfo.cs

```diff
@@ -39,10 +39,16 @@ public string UserId
         public string ServiceUrl { get; set; }
 
         /// <summary>
-        /// Gets or sets a value indicating whether the user is opted in to pairups
+        /// Gets or sets dictionary mapping team channel id to user's opt-in status for that team
         /// </summary>
         [JsonProperty("optedIn")]
-        public bool OptedIn { get; set; }
+        public IDictionary<string, bool> OptedIn { get; set; }
```

Comment:
What happens in the upgrade scenario from v1->v2?

Reply:
In the current implementation, since the `OptedIn` dictionary is only populated upon addition to a team, the only way to make sure the database accurately reflects the teams and users is to uninstall the old Icebreaker from every team and reinstall the new one. Combined with the other features, the database will probably also have to be reset, but the only information that would really be lost is opt in status.